lib/ServerTlsContext.php

```diff
@@ -160,7 +161,7 @@ public function withCiphers(string $ciphers = null): self {
     /**
      * @return string List of ciphers in OpenSSL's format (colon separated).
      */
-    public function getCiphers(): string {
+    public function getCiphers() {
         return $this->ciphers;
```

Comment:
This needs the same fix as `lib/ClientTlsContext.php`

Reply:
Ah, totally missed that :D These `*TlsContext` classes could clearly make use of a common abstract ancestor. At least these tests will ease that refactor later :)